config/sidekiq.yml

```diff
@@ -1,9 +1,10 @@
 :max_retries: 5
 
 :queues:
-  - [email_daily_alerts, 4]
+  - [email_alerts, 4]
```

Comment:
schedule.yml says `daily_alerts`, this says `email_alerts`. I think they should be the same.